core/src/com/unciv/logic/map/TileMap.kt

```diff
@@ -6,32 +6,29 @@ import com.unciv.logic.GameInfo
 import com.unciv.logic.HexMath
 import com.unciv.logic.civilization.CivilizationInfo
 import com.unciv.models.ruleset.Ruleset
+import kotlin.math.abs
 
 class TileMap {
 
     @Transient lateinit var gameInfo: GameInfo
     @Transient var tileMatrix = ArrayList<ArrayList<TileInfo?>>() // this works several times faster than a hashmap, the performance difference is really astounding
     @Transient var leftX = 0
     @Transient var bottomY = 0
+    @delegate:Transient val maxLatitude: Float by lazy { values.map { abs(it.latitude) }.max() ?: 0f }
```

Comment:
Instead of using ?: 0f, you could "if values.isEmpty 0f else ..."
It's clearer what the alternative is if it's spelled out =)

Reply:
I'm applying the change, but here I think it really makes it less readable.